package/MDAnalysis/analysis/density.py

```diff
@@ -1194,6 +1194,8 @@ def notwithin_coordinates_factory(universe, sel1, sel2, cutoff,
 
 
     .. deprecated:: 1.0.0
+    :func: `notwithin_coordinates_factory` is no longer supported and will be removed in 2.0.0. as part
```

Comment:
Two things here:

 - As far as I'm aware, there shouldn't be a space between :func: and `notwithin_coordinates_factory`.
  - The sentence seems to exceed the PEP8 80 character limit.

Reply:
That was sloppy of me, thanks :+1: